pyproject.toml

```diff
@@ -4,7 +4,7 @@ build-backend = "poetry.core.masonry.api"
 
 [tool.poetry]
 name = "intelligence-layer"
-version = "7.1.0"
+version = "7.2.0"
```

Comment:
Thank you for thinking about this, but we will do this ourselves when we do the next release